### ISSUE TITLE
Scrypted crashes in irregular intervals with TypeError: Cannot convert undefined or null to object (on Synology / Docker) 

### ISSUE THREAD

**Issue by manraf** [koush/scrypted#1231]
**Describe the bug**
I am running Scrypted in a docker container on a Synology NAS (DS 918+). It was setup according to the official documentation on https://github.com/koush/scrypted/wiki/Installation:-Docker-Synology-NAS with the current available "latest" image. I have three cameras running through it using the Synology and HomeKit plugins. 
In irregular intervals (27 times within 12 hours) it crashes with the following error:
```
TypeError: Cannot convert undefined or null to object
    at ServerResponse.removeHeader (node:_http_outgoing:799:30)
    at write (/server/node_modules/finalhandler/index.js:282:9)
    at send (/server/node_modules/finalhandler/index.js:306:5)
    at /server/node_modules/finalhandler/index.js:133:5
    at /server/node_modules/express/lib/router/index.js:646:15
    at next (/server/node_modules/express/lib/router/index.js:265:14)
    at /server/node_modules/@scrypted/server/src/scrypted-server-main.ts:277:9
    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/server/node_modules/express/lib/router/index.js:328:13)
    at /server/node_modules/express/lib/router/index.js:286:9
```

**To Reproduce**
The error cannot be reproduced, in fact, the described 27 crashes within 12 hours occured during the night, in which no interaction with the cams happened.

**Expected behavior**
No crashes or a more informative error message.

**Comment by zSprawl** [comment]
I'm seeing a similar issue. One previously stable cameras are going up and down every 10-15 minutes or so. Best I can tell is that it seemed to happen after a recent update of the rebroadcast component. I've tried using both Scrypted and FFmpeg settings without much luck.

**Comment by koush** [comment]
I had a repro once but I have no idea what is causing it. seems to be in the http server internals and not anything related to scrypted. I haven't updated those dependencies lately, so I'm at a loss.

**Comment by koush** [comment]
I think this might be fixed in v0.80.0
